worldedit-core/src/main/java/com/sk89q/worldedit/math/BlockVector3.java

```diff
@@ -35,27 +35,16 @@
     public static final BlockVector3 UNIT_X = new BlockVector3(1, 0, 0);
     public static final BlockVector3 UNIT_Y = new BlockVector3(0, 1, 0);
     public static final BlockVector3 UNIT_Z = new BlockVector3(0, 0, 1);
+    public static final BlockVector3 UNIT_MINUS_X = new BlockVector3(-1, 0, 0);
+    public static final BlockVector3 UNIT_MINUS_Y = new BlockVector3(0, -1, 0);
+    public static final BlockVector3 UNIT_MINUS_Z = new BlockVector3(0, 0, -1);
```

Comment:
does it make sense to add the negative units to Vector as well?

Reply:
I didn't see anywhere need the other ones.